common/src/stack/report-system/command/report/system/tests/test_partitions.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+import math
+from collections import namedtuple, defaultdict
+from stack import api
+from stack.commands.report.system.fixtures.disk_partition_check import verify_disk
+
+@pytest.mark.parametrize('hostname',
+list(set([host['host'] for host in api.Call('list host partition')]))
```

Comment:
I believe this can be simplified to `set(host['host'] for host in api.Call('list host partition'))`. The code inside the set is a [generator expression](https://www.python.org/dev/peps/pep-0289/) which ends up being more efficient than creating a list in memory that will be thrown away.

I also believe `pytest.mark.parametrize` can iterate over the `set` without it being turned back into a `list`.